_test/tests/inc/remote.test.php

```diff
@@ -332,6 +345,19 @@ function test_pluginCallMethods() {
         $this->assertEquals($remoteApi->call('plugin.testplugin.methodString'), 'success');
     }
 
+    /**
+     * @expectedException dokuwiki\Remote\RemoteException
```

Comment:
Could we also check the error code here?

Reply:
Sure, then I can rewrite this as a try catch rather than the comment annotation.